### DISCUSSION TITLE
Granular config for GitactionBoard

### DISCUSSION THREAD

**Discussion by sumanmaity1234** [otto-de/gitactionboard#117] [Ideas]
Currently, we have lots of environment variable to configure backend. Till now it works most of the cases, but certain cases where granular config is required, currently it's not possible.

Main idea is, user should be able to define different **repo names** for different features (workflow, codescan, secretsscan etc). We can also introduce granular config for **notifications** as well.

**Comment by cshruti00** [comment]
I am thinking to change the env variable ENABLE_GITHUB_SECRETS_SCAN_ALERTS_MONITORING, ENABLE_GITHUB_CODE_SCAN_ALERTS_MONITORING to accept repo_names. 

I will work on this based on the above idea.

Notification to MS teams, I will check and get back.
